containers/ecr-viewer/src/app/tests/services/evaluateFhirDataServices.test.ts

```diff
@@ -427,6 +430,42 @@ Home: 123-456-6909`,
     });
   });
 
+  describe("Evaluate Hospital Encounter Data", ()=>{
+    it("should return unavailable data when no Admission Diagnosis or Discharge diagnosis are found", () => {
+      const bundle: Bundle = {
+        resourceType: "Bundle",
+        type: "document",
+        entry: [],
+      };
+
+      expect(evaluateHospitalEncounterData(bundle)).toEqual({"availableData": [], "unavailableData": [{"table": true, "title": "Hospital Admission Diagnosis", "value": undefined}, {"table": true, "title": "Hospital Discharge Diagnosis", "value": undefined}]});
+    });
+
+    it("should return Hospital Encounter Data and match snapshot", () => {
+      const actual = evaluateHospitalEncounterData(
+          BundlePatientWithHospitalEncounterData as unknown as Bundle,
+      );
+
+      expect(actual).toMatchSnapshot();
+    });
+
+    it("A bundle with only Admission Diagnosis returns that data and matches snapshot", () => {
+      const actual = evaluateHospitalEncounterData(
+          BundleHospitalEncounterOnlyAdmissionDx as unknown as Bundle,
+      );
+
+      expect(actual).toMatchSnapshot();
+    });
+
+    it("A bundle with only Discharge Diagnosis returns that data and matches snapshot", () => {
+      const actual = evaluateHospitalEncounterData(
+          BundleHospitalEncounterOnlyDischargeDx as unknown as Bundle,
```

Comment:
instead of having three new bundles here, can we use one and add/remove the admission/discharge references as needed? (there are some examples of this elsewhere - maybe labs?)

Reply:
could we also add some sanity check that actual is what we want besides the snapshot? (I can see someone (perhaps, me 😬 ) blindly updating snapshots at some point and missing something)

Reply:
I'll take a look

Reply:
Added additional checks and also a somewhat funky system to push necessary admission and discharge dx data into a fhir bundle for testing